### DISCUSSION TITLE
so-rule This should not happn

### DISCUSSION THREAD

**Discussion by intsec** [Security-Onion-Solutions/securityonion#6985] [Unsupported Versions]
When working with the so-rule command I am getting this every time I run it.

(This should not happen, the system is in an error state if you see this message.)

More than one manager-type pillar exists, minion id's listed below:
  ", [, ', /, o, p, t, /, s, o, /, s, a, l, t, s, t, a, c, k, /, l, o, c, a, l, /, p, i, l, l, a, r, /, m, i, n, i, o, n, s, /, a, w, s, _, s, t, a, n, d, a, l, o, n, e, ., s, l, s, ', ,,  , ', /, o, p, t, /, s, o, /, s, a, l, t, s, t, a, c, k, /, l, o, c, a, l, /, p, i, l, l, a, r, /, m, i, n, i, o, n, s, /, i, n, t, s, e, c, -, i, d, s, _, s, t, a, n, d, a, l, o, n, e, ., s, l, s, ', ], "

I just started using this so not sure when this started happening.
Thanks
Eric

**Comment by dougburks** [comment]
It seems like you have multiple `*_standalone.sls` files in `/opt/so/saltstack/local/pillar/minions/` and this shouldn't happen under normal conditions. 

Did you modify these files? 

Did you re-rerun Setup with a different hostname?

**Reply by intsec** [reply]
Hi Doug, thanks for the response.  I do not remember at the time if I did modify these.  Would I be able to tell somehow by looking at them?  As for the setup, I did not change my hostname.

**Reply by dougburks** [reply]
There should be one `*_standalone.sls` file in that directory that matches your hostname. For the `*_standalone.sls` file that doesn't match your hostname, move it to your home directory just in case you need it.
